flags.go

```diff
@@ -41,9 +46,16 @@ func (f *KubeHTTPProxyFlags) Parse() error {
 	flag.StringVar(&f.Frontend.Address, "frontend-addr", "0.0.0.0", "TCP address to listen on")
 	flag.IntVar(&f.Frontend.Port, "frontend-port", 80, "TCP address to listen on")
 
+	flag.BoolVar(&f.Frontend.Watch, "frontend-watch", false, "watch for Kubernetes frontend updates")
+	flag.StringVar(&f.Frontend.Namespace, "frontend-namespace", "", "name of Kubernetes frontend namespace")
+	flag.StringVar(&f.Frontend.Service, "frontend-service", "", "name of Kubernetes frontend service")
+	flag.StringVar(&f.Frontend.PortName, "frontend-portname", "http", "name of frontend port")
+
+	flag.BoolVar(&f.Backend.Watch, "backend-watch", false, "watch for Kubernetes backend updates")
 	flag.StringVar(&f.Backend.Namespace, "backend-namespace", "", "name of Kubernetes backend namespace")
 	flag.StringVar(&f.Backend.Service, "backend-service", "", "name of Kubernetes backend service")
-	flag.StringVar(&f.Backend.Port, "backend-port", "http", "name of backend port")
+	flag.StringVar(&f.Backend.PortName, "backend-portname", "http", "name of backend port")
```

Comment:
@martin-helmich yes, this is a good idea.

Reply:
@martin-helmich 

Yes, this is a good idea. I also want to be consistent between flag names for backend and frontend. There is already `-port` flag for the port number to run frontend on so I have chosen `-portname` flag for the port name. I guess the best idea would be to support both flags for the backend (`backend-portname` and `backend-port). In case if `backend-port` is used we can throw a warning message that `backend-port` has  been deprecated in favor of `backend-portname` and will be removed in future versions.

What do you think?

Reply:
>In case if backend-port is used we can throw a warning message that backend-port has been deprecated in favor of backend-portname and will be removed in future versions.

Sounds good to me! :+1:

Reply:
@martin-helmich so this is done :)